spec/controllers/curation_concerns/collections_controller_spec.rb

```diff
@@ -245,11 +245,20 @@
   end
 
   describe '#edit' do
-    before { sign_in user }
+    let!(:my_collection) { create(:collection, user: user) }
+
+    before do
+      # We expect to not see this collection in the list
+      create(:collection, :public)
+      sign_in user
+    end
 
-    it 'does not show flash' do
+    it 'is successful' do
       get :edit, params: { id: collection }
       expect(flash[:notice]).to be_nil
+      # a list of collections I can add items to:
+      expect(assigns[:user_collections].map(&:id)).to match_array [collection.id,
```

Comment:
The collection created on 252 does not show up. The two that match are on line 25 and on line 248.